comictaggerlib/settingswindow.py

```diff
@@ -235,6 +234,100 @@ def disconnect_signals(self) -> None:
         self.twLiteralReplacements.cellChanged.disconnect()
         self.twValueReplacements.cellChanged.disconnect()
 
+        self.sources: dict = {}
+        self.generate_source_option_tabs()
+
+    def generate_source_option_tabs(self) -> None:
+        def format_internal_name(int_name: str = "") -> str:
+            # Presume talker_<name>_<nm>
+            int_name_split = int_name.split("_")
+            del int_name_split[0:3]
+            int_name_split[0] = int_name_split[0].capitalize()
+            new_name = " ".join(int_name_split)
+            return new_name
+
+        # Add source sub tabs to Comic Sources tab
+        for talker_id, talker_obj in self.talkers.items():
+            # Add source to general tab dropdown list
+            self.cobxInfoSource.addItem(talker_obj.name, talker_id)
+
+            # Use a dict to make a var name from var
+            source_info = {}
+            tab_name = talker_id
+            source_info[tab_name] = {"tab": QtWidgets.QWidget(), "widgets": {}}
+            layout_grid = QtWidgets.QGridLayout()
+            row = 0
+
+            full_talker_name = "talker_" + talker_id
```

Comment:
This variable name is longer than the content and `full_talker_name` sounds like it's the display name.
Using `f"talker_{talker_id}"` Mae's it much easier to see what it is